src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp

```diff
@@ -35,21 +38,28 @@ enum Creatures
 
 Position const PosSummon[3] =
 {
-    {3267.9f, -3172.1f, 297.42f, 0.94f},
-    {3253.2f, -3132.3f, 297.42f, 0},
-    {3308.3f, -3185.8f, 297.42f, 1.58f},
+    { 3267.9f, -3172.1f, 297.42f, 0.94f },
+    { 3253.2f, -3132.3f, 297.42f, 0 },
+    { 3308.3f, -3185.8f, 297.42f, 1.58f },
 };
 
 enum Events
 {
-    EVENT_WOUND     = 1,
+    EVENT_WOUND = 1,
     EVENT_ENRAGE,
     EVENT_DECIMATE,
     EVENT_BERSERK,
     EVENT_SUMMON,
+    EVENT_SEARCH_ZOMBIE,
+    EVENT_KILL_ZOMBIE,
 };
 
-#define EMOTE_NEARBY    " spots a nearby zombie to devour!"
+#define EMOTE_NEARBY    "Gluth spots a nearby zombie to devour!"
+#define EMOTE_DECIMATE  "Gluth decimates all nearby flesh!"
+#define EMOTE_ENRAGE    "Gluth becomes enraged!"
+
+#define SECONDS *1000
+#define MINUTES *60000
```

Comment:
User defined literals are already supported in c++11, ( this will support literals like `13_s` or `2_min` becaus ethe underscore is forced) - only supported in gcc, clang and msvc 2015.
C++14 introduces duration literals which don't need an underscore anymore `13s` or `2min`.

But for now you can try to use the [TaskScheduler](https://gist.github.com/Naios/6d143538abdc3d32fd2b) which ships with some nice time typedefs like `Seconds(13), Minutes(2)` so you can drop your custom defines:

```c++
scheduler.Schedule(Minutes(2) + Seconds(25), [this](TaskContext context)
{
    // do something
});
```